app/src/main/res/drawable/thumbnail_gradient.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <gradient
+    android:angle="270"
+    android:centerColor="#00ffffff"
+    android:endColor="#ffff"
```

Comment:
Always use 6 or 8 characters hex color code. Change this to `#ffffff`

Reply:
Done.